### PR TITLE
chore: user management widget RELEASE

### PR DESCRIPTION
**Pull request by nirgur**
## Related Issues

related https://github.com/descope/etc/issues/4634

### CODE REVIEW

**Comment by dorsha** [comment]
Very nice job!

**Comment by nirgur** [comment]
> Very nice job!

Thanks!

File: packages/user-management-widget/src/lib/mixins/loggerMixin/loggerMixin.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export const loggerMixin = createSingletonMixin(
+  <T extends CustomElementConstructor>(superclass: T) =>
+    class LoggerMixinClass extends superclass {
+      #logger: Logger = this.#wrapLogger(console);
+
+      #wrapLogger(logger: Partial<Logger>) {
+        return logLevels.reduce((acc, logLevel) => {
+          acc[logLevel] = (...args: any[]) => {
+            // this.dispatchEvent(new CustomEvent(INTERNAL_LOG_EVENTS[logLevel], { detail: args }));
```
Comment (tomerlichtash) [inline packages/user-management-widget/src/lib/mixins/loggerMixin/loggerMixin.ts:16]: Remove

File: packages/user-management-widget/src/lib/mixins/notificationsMixin/notificationsMixin.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            | 'top-start'
+            | 'top-center'
+            | 'top-end'
+            | 'middle'
+            | 'bottom-start'
+            | 'bottom-center'
+            | 'bottom-end'
+            | 'bottom-stretch';
+          size: 'xs' | 'sm' | 'md' | 'lg';
+          icon?: 'success' | 'error';
```
Comment (tomerlichtash) [inline packages/user-management-widget/src/lib/mixins/notificationsMixin/notificationsMixin.ts:51]: Remove

Comment (tomerlichtash) [reply packages/user-management-widget/src/lib/mixins/notificationsMixin/notificationsMixin.ts:51]: Add bordered if you want

**Review by tomerlichtash — APPROVED** [review]
Hamazing!

**Pull request merged**